[SB] accept only destination for forward_to (#17430)
Azure/azure-sdk-for-python: sdk/servicebus/azure-servicebus/CHANGELOG.md

## 7.1.1 (Unreleased)

**New Features**

> * Updated `forward_to` parameter in `create_queue` and `create_subscription` methods on `ServiceBusAdministrationClient` to accept entities as well, rather than only full paths. In the case that an entity is passed in, it is assumed that the entity exists within the namespace specified by the `fully_qualified_namespace` property of the `ServiceBusAdministrationClient`.

Reviewer: nit: what you described is technically correct, but the namespace is specified when creating the `ServiceBusAdministrationClient`, the property is populated during the creation process. so what I'm thinking of is like the following:

also I see you updated `forward_dead_lettered_messages_to` but it's not mentioned here
```suggestion
* Updated `forward_to` and `forward_dead_lettered_messages_to` parameters in `create_queue` and `create_subscription` methods on `ServiceBusAdministrationClient` to accept entities as well, rather than only full paths. In the case that an entity is passed in, it is assumed that the entity exists within the same namespace used for constructing the `ServiceBusAdministrationClient`.
```
